safedelete/tests/test_soft_delete.py

```diff
@@ -161,7 +187,8 @@ def test_update_or_create_with_unique_together_constraint(self):
         obj, created = UniqueTogetherSoftDeleteModel.objects.update_or_create(name='thor', team='avengers')
         self.assertEqual(obj.name, 'thor')
         self.assertEqual(obj.team, 'avengers')
-        self.assertEqual(created, False)
+        self.assertFalse(created)
+        self.assertTrue(UniqueTogetherSoftDeleteModel.has_unique_fields())
```

Comment:
Useful explicit test; perhaps I should add entire new test
```
def test_model_has_unique_fields():
        self.assertFalse(UniqueSoftDeleteModel.has_unique_fields())
        self.assertTrue(UniqueTogetherSoftDeleteModel.has_unique_fields())
        self.assertFalse(UniqueConstraintSoftDeleteModel.has_unique_fields())
```

Reply:
Yes this would be better :+1: .